api/src/routes/applications/#id/delete.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, Router } from "express";
+import { User, Application, Member } from "@fosscord/util";
+import { route } from "@fosscord/api";
+
+const router: Router = Router();
+
+router.post("/", route({}), async (req: Request, res: Response) => {
+	await Promise.all([
+		Application.delete({ id: req.params.id }),
+		User.delete({ id: req.params.id }),
+		Member.delete({ id: req.params.id })
+	]);
```

Comment:
Wouldn't this allow *anyone* to delete *any* application if they knew the app ID?

Reply:
i will fix this by adding a check to only the owner of the application can delete